OWNERS_ALIASES

```diff
@@ -7,6 +7,7 @@ aliases:
     - prydonius
     - kow3ns
     - janetkuo
+    - barney-s
   apps-application-reviewers:
```

Comment:
Do you want to add yourself as reviewers as well?

Reply:
done